swri_transform_util/swri_transform_util/origin_manager.py

```diff
@@ -127,7 +127,7 @@ def __init__(self, node, local_xy_frame, local_xy_frame_identity=None):
             local_xy_frame_identity = local_xy_frame + "__identity"
         self.local_xy_frame_identity = local_xy_frame_identity
         qos = rclpy.node.QoSProfile(depth=1,
-                                    durability=rclpy.qos.QoSDurabilityPolicy.RMW_QOS_POLICY_DURABILITY_TRANSIENT_LOCAL)
+                                    durability=rclpy.qos.QoSDurabilityPolicy.TRANSIENT_LOCAL)
```

Comment:
:pray: thank you for fixing this! The warning is really annoying

Reply:
I don't abide build warnings haha